### PR TITLE
V4-java-SDK-ppplmer 96500 update query parameters-PART B

### PR DESCRIPTION
**Pull request by saikgupta**
This is the Pull request for the ticket PPPLMER-96500 https://engineering.paypalcorp.com/jira/browse/PPPLMER-96500. Please review the changes done for missing query parameters for below resources:

1.List Prepaid Card.
2.List Paper Check.
3.List Payment.
4. List Transfer Method.

### CODE REVIEW

File: src/test/java/com/hyperwallet/clientsdk/model/HyperwalletPaymentListOptionsTest.java [hunk 1]
```diff
@@ -1,15 +1,19 @@
 package com.hyperwallet.clientsdk.model;
 
+import java.text.ParseException;
```
Comment (ramahalingam) [inline src/test/java/com/hyperwallet/clientsdk/model/HyperwalletPaymentListOptionsTest.java:3]: Pls remove all the imports here.

**Review by vkorotchenko — APPROVED** [review]

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPaperCheck.java [hunk 2]
```diff
@@ -856,4 +850,12 @@ public HyperwalletPaperCheck clearLinks() {
         this.links = null;
         return this;
     }
+
+    public enum ShippingMethod {STANDARD, EXPEDITED}
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPaperCheck.java:854]: These enums should be at the top.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPrepaidCard.java [hunk 3]
```diff
@@ -356,4 +346,14 @@ public HyperwalletPrepaidCard clearReplacementReason() {
         this.replacementReason = null;
         return this;
     }
+
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPrepaidCard.java:349]: enums should be at the top

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java [hunk 4]
```diff
@@ -12,7 +12,9 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletReceipt {
 
-    public static enum Entry { DEBIT, CREDIT };
+    public static enum Entry {DEBIT, CREDIT}
+
+    ;
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java:17]: Delete this semicolon.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java [hunk 5]
```diff
@@ -12,7 +12,9 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletReceipt {
 
-    public static enum Entry { DEBIT, CREDIT };
+    public static enum Entry {DEBIT, CREDIT}
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java:15]: delete the static keyword

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java [hunk 6]
```diff
@@ -12,7 +12,9 @@
...
+
+    ;
 
     public static enum Type {
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletReceipt.java:19]: delete the static keyword

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletTransferMethod.java [hunk 7]
```diff
@@ -1415,4 +1402,12 @@ public HyperwalletTransferMethod clearLinks() {
         this.links = null;
         return this;
     }
+
+    public enum Type {BANK_ACCOUNT, WIRE_ACCOUNT, PREPAID_CARD, BANK_CARD, PAPER_CHECK, PAYPAL_ACCOUNT, VENMO_ACCOUNT}
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletTransferMethod.java:1406]: enums should be at the top.

**Review by ramahalingam — COMMENTED** [review]
Pls delete the binary file in the comment and fix the other comments.

**Review by ramahalingam — APPROVED** [review]

**Review by arao6 — APPROVED** [review]

**Comment by Rrathinasabapath** [comment]
Changes were merged into master

**Pull request merged**